receiver/httpcheckreceiver/README.md

```diff
@@ -37,6 +37,7 @@ Each target has the following properties:
 
 - `endpoint` (required): the URL to be monitored
 - `method` (optional, default: `GET`): The HTTP method used to call the endpoint
+- `substring` (optional, default: ""): If set, the receiver will emit metrics based on whether the response body matches this string.
```

Comment:
As per the description of this option, `body` seems more descriptive

```suggestion
- `body` (optional, default: ""): If set, the receiver will emit metrics based on whether the response body matches this string.
```

Reply:
I agree `body` is much better name for both the configuration property and the metric name.

I would also expect the description to specify whether it's a regex match or exact match.